lib/alertmanager_api/silence.py

```diff
@@ -0,0 +1,198 @@
+import logging
+import requests
+from exceptions.missing_mandatory_param_error import MissingMandatoryParamError
+from requests.auth import HTTPBasicAuth
+from requests.exceptions import RequestException
+from alertmanager_api.silence_details import SilenceDetailsHandler
+
+
+class AlertManagerAPI:
```

Comment:
Don't think you get anything out of making this a class - just convert your CRUD methods into standalone functions which take `AlertmanagerAccount` as an extra input - allows us to call the function directly from the actions/sensor layer without needing to create an extra middle `workflows` layer